runtime/dancebox/tests/integration_test.rs

```diff
@@ -67,6 +67,13 @@ fn genesis_balances() {
             // Alice gets 10k extra tokens for her mapping deposit
             (AccountId::from(ALICE), 210_000 * UNIT),
             (AccountId::from(BOB), 100_000 * UNIT),
+            (AccountId::from(CHARLIE), 100_000 * UNIT),
+            (AccountId::from(DAVE), 100_000 * UNIT),
+        ])
+        .with_collators(vec![
+            // Remove ALICE and BOB from collators
+            (AccountId::from(CHARLIE), 100 * UNIT),
+            (AccountId::from(DAVE), 100 * UNIT),
```

Comment:
This was needed because if ALICE is a collator, her free balance is 1 unit less than expected and this test fails.